requester/src/requester/download.py

```diff
@@ -8,24 +8,75 @@
 from loguru import logger
 
 GameID = typing.NewType("GameID", int)
+PatchVersion = typing.NewType("PatchVersion", typing.Tuple[str, str])
 
-CALLS_PER_SECOND: int = 1
+CALLS_PER_SECOND = 1
 DEFAULT_RETRY_ATTEMPTS = (0, 1, 2, 5, 10, 30)
 
 
-class TooManyTriesError(Exception):
+@dataclasses.dataclass(frozen=True)
+class DownloadResult:
+    game_id: GameID
+
+
+@dataclasses.dataclass(frozen=True)
+class DownloadedGame(DownloadResult):
+    data: typing.Dict[str, typing.Any]
```

Comment:
In recent Python versions you can use
`data: dict[str, typing.Any]`
instead of 
`data: typing.Dict[str, typing.Any]`